Cargo.toml

```diff
@@ -91,6 +92,7 @@ futures = "0.3.17"
 maplit = "1.0.2"
 num_cpus = "1.13.0"
 simple_logger = "2.1.0"
+regex = "1.11.1"
```

Comment:
This is added only for tests. I decided to use regex instead of complex semver parsing logic in a test

Reply:
https://docs.rs/semver/latest/semver/ should be the first thing to reach for when doing version parsing in rust I think, no?